docs/guides/blocks/formulas.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Formulas
+---
+
+# Add custom formulas
+
+Formulas are one of the most basic building blocks in Coda; used to calculate values, filter tables, and so much more. Coda provides a wide array of [built-in formulas][formulas], and using Packs you can add your own custom formulas. Once your Pack is installed in a doc you can use those custom formulas anywhere, intermingling them with built-in formulas or those from other Packs.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Structure of a formula
+
+TODO
+
+
+## Naming
+
+The name of a formula can only contain the letters, numbers, and underscores. By convention these names should by written in upper camel case, like `DoSomethingCool`.
```

Comment:
Optional, could mention that the character limitations are based the allowed characters in the Coda Formula Language.

Reply:
Done